app/src/main/java/com/kickstarter/ui/activities/WebViewActivity.java

```diff
@@ -47,12 +46,19 @@ protected void onCreate(final @Nullable Bundle savedInstanceState) {
       .subscribe(webView::loadUrl);
   }
 
-  @OnClick(R.id.back_button)
```

Comment:
this `@OnClick(R.id.back_button)` was redundant since this method exists in our base `KSToolbar`